action/action-util/src/main/java/com/powsybl/action/util/AbstractScalable.java

```diff
@@ -21,14 +23,71 @@ protected AbstractScalable() {
     }
 
     @Override
+    public List<Injection> filterInjections(Network network, List<String> notFound) {
+        List<Injection> injections = new ArrayList<>();
+        filterInjections(network, injections, notFound);
+        return injections;
+    }
+
+    @Override
+    public List<Injection> filterInjections(Network network) {
+        return filterInjections(network, null);
+    }
+
+
+    /**
+     * @deprecated listGenerators should be replaced by filterInjections
+     */
+    @Deprecated
     public List<Generator> listGenerators(Network n, List<String> notFoundGenerators) {
         List<Generator> generators = new ArrayList<>();
         listGenerators(n, generators, notFoundGenerators);
         return generators;
     }
 
-    @Override
+    /**
+     * @deprecated listGenerators should be replaced by filterInjections
+     */
+    @Deprecated
     public List<Generator> listGenerators(Network n) {
         return listGenerators(n, null);
     }
+
+    /**
+     * @deprecated listGenerators should be replaced by filterInjections
+     */
+    @Deprecated
+    public void listGenerators(Network network, List<Generator> generators, List<String> notFoundGenerators) {
```

Comment:
Could you check this method is correctly implemented? It seems to me that notFoundgenerators can contain loads.

Reply:
In fact, not found generators may contain load IDs. It is a way to inform the user that it cannot find a generator that has as ID the ID of another kind of injection.
That's how we designed it, but as you see it is put as deprecated, so it is supposed to be removed at the end. You would prefer that we check it does not exist at all in the network ? Log an error or throw an exception if it is an object of other type ?

Reply:
As this method will be removed soon and  is not expected to be used mixed with loads, we can let t as it is.